pkg/internal/checkup/executor/executor.go

```diff
@@ -43,9 +43,9 @@ type Executor struct {
 	namespace                        string
 	vmiUsername                      string
 	vmiPassword                      string
-	vmiEastNICPCIAddress             string
```

Comment:
From the first commit message:
> Now that both the traffic generator and DPDK are VMIs, the PCI-Address

I suggest replacing "DPDK" with VMI under test or something similar.

Same for the second commit message.

Reply:
DONE, also for the PR desc